### PR TITLE
Feat/add tweet

### PR DESCRIPTION
**Pull request by ritachien**
add a tweet

### CODE REVIEW

**Comment by ritachien** [comment]
本來看到 feat: add followship seed file 以為不小心 merge 到別條分支了，所以刪除分支想重發一次。後來發現那個是因為 git pull 才出現的 ......就直接復原 pull request 了!

File: controllers/tweet-controller.js [hunk 1]
```diff
@@ -13,6 +14,31 @@ const tweetController = {
...
+    const { description } = req.body
+    if (!userId || !description) {
+      return res.status(400).json({
+        status: 'error',
+        message: 'userId and description required'
+      })
+    }
+
+    if (description.length > 140) {
+      return res.status(401).json({
```
Comment (rubylo718) [inline controllers/tweet-controller.js:29]: please consider to use an appropriate status code instead of 401(unauthorized)

**Review by rubylo718 — COMMENTED** [review]
1. Please consider to use an appropriate status code instead of 401(unauthorized) at tweet-controller line 29.
3. As discussed, please make sure the output data structure meets the request in test file.

**Comment by ritachien** [comment]
new commit updated!

File: controllers/tweet-controller.js [hunk 2]
```diff
@@ -13,6 +14,27 @@ const tweetController = {
...
+
+    if (description.length > 140) {
+      return res.status(400).json({
+        status: 'error',
+        message: 'tweet should be within 140 characters'
+      })
+    }
+
+    const tweet = await Tweet.create({ userId, description })
+    return res.status(200).json(tweet.toJSON())
```
Comment (rubylo718) [inline controllers/tweet-controller.js]: The "res.json()" method sends a JSON response. I think we don't need to use .toJSON() to convert the object before sending it to .json() method as its parameter.

Comment (ritachien) [reply controllers/tweet-controller.js]: Oh~ I forgot that one. Used to do this the whole semester 3 😂

**Review by rubylo718 — APPROVED** [review]
ya we just learnt that last week lol. keep going forward!

**Pull request merged**